app/views/admin/companies/edit.html.erb

```diff
@@ -8,11 +21,13 @@
     <%= f.number_field :hire_count, in: 0..500, step: 1, value: @company.hire_count %><br>
     <%= f.label(:hire_count, "Number of people hiring") %>
   </div>
-  <div class="form-group">
-    <%= f.hidden_field :status %>
-  </div>
-    <%= f.submit "Submit" %>
+  <%= f.hidden_field :status %>
+  <%= f.hidden_field :name %>
+  <%= f.hidden_field :title %>
+  <%= f.hidden_field :email %>
```

Comment:
This is on the admin companies edit page.  Since the fields are hidden, they shouldn't be able to change anything.  The reason it's there, though, is because when you try to update a company, you have to have all the current status, name, title and email due to the way it's setup in the model right now.  Perhaps if we remove the validation for name, status, email and title, we won't need to have these hidden fields here.